tests/nnstreamer_filter_tvm/unittest_filter_tvm.cc

```diff
@@ -485,6 +488,7 @@ main (int argc, char **argv)
   }
 
   gst_init (&argc, &argv);
+  gst_tensor_alloc_init (127UL);
```

Comment:
How about in tvm subplugin's init function?

Reply:
That sounds right. Updated!!